src/pages/HomePage/mocks/index.ts

```diff
@@ -1,11 +1,21 @@
 import { rest } from "msw"
 
-import { ENDPOINTS } from "@constants/endPoints"
-
-import { mockData } from "./mockData"
+import { mockData, mockDataReleased, mockDataSortView } from "./mockData"
 
 const allProjectHandlers = [
-  rest.get(ENDPOINTS.GET_ALL_PROJECTS, (_, res, ctx) => {
+  rest.get(
+    "http://3.39.156.144:8080/api/v1/projects?isReleased=true",
+    (_, res, ctx) => {
+      return res(ctx.status(200), ctx.json(mockDataReleased))
+    },
+  ),
```

Comment:
굳이 수정 안하셔도 됩니다~

```suggestion
  rest.get(
    "http://3.39.156.144:8080/api/v1/projects",
    (req, res, ctx) => {
      const { isReleased } = req.params
      // ...
      return res(ctx.status(200), ctx.json(mockDataReleased))
    },
  ),
```
msw에서 쿼리 스트링 가져오는 방법입니다~!
나중에 필요하시면 이렇게도 사용해보세요!

Reply:
오오 감사합니다!